meta-mbl-distro/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -42,6 +42,27 @@ python __anonymous() {
         d.appendVar("FSTAB_LINES", fstab_line)
 }
 
+# Write the partition sizes and offsets to files which will be placed in /config/factory
+python __anonymous() {
```

Comment:
Can we create a new task and execute it before do _install?
Creating the WORKDIR at parse time is a bit hacky to me.

Reply:
I think using `do_compile` would be fine as well. But if `do_compile` has already been defined as a shell function then it would be easier to create a new task.